app/models/user.rb

```diff
@@ -38,9 +38,10 @@ module Frequency
     c.validates_format_of_login_field_options = { with: Authlogic::Regex::LOGIN, message: I18n.t('error_messages.login_invalid', default: "can only consist of alphabets, numbers, underscore '_', and hyphen '-'.") }
   end
 
-  has_attached_file :photo, styles: { thumb: '200x200#', medium: '500x500#', large: '800x800#' },
-                                    url: '/system/profile/photos/:id/:style/:basename.:extension'
-  #:path => ":rails_root/public/system/images/photos/:id/:style/:basename.:extension"
+  has_attached_file :photo,
+                    styles: { thumb: '200x200#', medium: '500x500#', large: '800x800#' },
+                    url: '/public/system/profile/photos/:id/:style/:basename.:extension',
+                    path: ':rails_root/public/system/public/system/profile/photos/:id/:style/:filename'
```

Comment:
Oh wait, the other way around. So the images you're finding are at `users` but we are pointing to `profile`?

Reply:
I had been working from this old image: https://storage.googleapis.com/publiclab-production/public/system/profile/photos/554/thumb/Bristol-539-7.jpg which currently works.